pages/03_text2image.py

```diff
@@ -0,0 +1,70 @@
+import streamlit as st
+
+from config import OpenAIClient
+
+# openai client
+client = OpenAIClient()
+
+# promptファイル
+PROMPT_FILE = "./prompts/03_createImage.md"
+
+def init_page():
+    """ ページ設定 """
+    st.set_page_config(
+        page_title = "text2image x llm"
+    )
+    st.title("text2image x llm")
+    if "chat_image_log" not in st.session_state:
+        st.session_state.chat_image_log = []
```

Comment:
- ミソはresponse_idをsession_stateで管理する、ですね
```suggestion
    if "chat_image_log" not in st.session_state:
        st.session_state.chat_image_log = []
    if "response_id" not in st.session_state:
        st.session_state.response_id = None
```